cycode/cli/files_collector/sca/sca_code_scanner.py

```diff
@@ -104,7 +103,7 @@ def try_restore_dependencies(
             restore_dependencies_document.content = ''
         else:
             is_monitor_action = context.obj['monitor']
-            project_path = context.params['paths'][0]
+            project_path = context.params['path'][0]
```

Comment:
Probably we want to handle both cases? In the case of `cycode scan repository` as I can see `path` will be right, but in the case of `cycode scan path` it will be `paths`. Pls check

Reply:
Added checks for both